LSN_7/home-work/main.js

```diff
@@ -0,0 +1,67 @@
+// Преобразование массива строк в массив их длин (использовать map):
+const words = ["apple", "banana", "cherry"];
+const newArr = words.map((e) =>  e.length)
+console.log(newArr);
+
+// Фильтрация массива чисел, оставляющая только четные числа (использовать filter):
+const numbers = [1, 2, 3, 4, 5];
+const arr = numbers.filter((e) => {
+    if(e % 2 === 0) {
+        console.log(e);
+    }
+})
+// Фильтрация массива строк, оставляющая только слова с буквой "a" (использовать filter и includes):
+const array = words.filter((e) => {
+    if(e.includes('a')) {
+        console.log(e);
+    }
+})
+// Поиск первого числа в массиве, которое больше 10 (использовать find):
+const number = [8, 12, 5, 20, 3];
+const num = number.find((e) => {
+    return e > 10
+})
+console.log(num);
+// Добавление элемента в конец массива (использовать push):
+const fruits1 = ["apple", "banana"];
+fruits1.push('eggplant')
```

Comment:
Use a consistent coding style. Double quotes everywhere if you use them
fruits1.push("eggplant")